src/triples.rs

```diff
@@ -1060,7 +1060,7 @@ impl SvnTypeChoice {
 }
 
 /// Collection of one or more cryptographic digests
-pub type DigestsType<'a> = Vec<Digest<'a>>;
+pub type DigestsType<'a> = Vec<Digest>;
```

Comment:
If we are dropping the lifetime from Digest, we can drop it from DigestsType, as well.

Reply:
Good point. I've removed it.